examples/main.tf

```diff
@@ -0,0 +1,74 @@
+provider "azurerm" {
+  version = "1.22.1"
```

Comment:
no example should be using a provider anywhere near this old

Reply:
This was used to remain inline with the version used for IDAM web apps. Assumed this was a version constraint. 

Update